README.md

```diff
@@ -13,6 +13,51 @@ These witnessed checkpoints can be consumed by clients that want protection agai
 
 Users wishing to run this should start with the [OmniWitness](./cmd/omniwitness/).
 
+## Importance of Witnesses
+
+Some important terminology:
+
+Term                | Definition
+--------------------|------------
+Log                 | A verifiable data structure that commits to a list of entries
+Append Only         | New entries added to a log can only be added to the end of the list
+Checkpoint          | A Checkpoint is a _signed_ commitment to the state of a Log at a given size
+Split View          | A split view is a situation where a Log presents two views of its data where one is not a prefix of the other
+Consistency Proof   | A crytographic proof between two checkpoints that proves one of them is an Append Only evolution of the other
+Witness             | Tracks one or more logs, verifying Consistency Proofs to ensure it cannot cross any Split Views
+
+Witnesses are a critical part of a secure transparency ecosystem by providing an anchor that guards against split view attacks by log operators.
+
+A log that could successfully perform a split view would be able to show one list of entries to users that rely on the data (e.g. malware), and a different list of data to the verifiers of this data (e.g. only good builds of software).
+
+This fundamentally undermines the main property of logs: _discoverability_.
+Discoverability ensures that any data that is relied upon by one user will eventually be discovered and verified by someone that is holding the claimant and the log to account.
+See https://transparency.dev/how-to-design-a-verifiable-system/ for information on the Claimant Model, which succinctly describes roles in a transparency ecosystem.
+
+
+A Witness keeps a single Checkpoint for each log in secure storage.
+On first initialization, this Checkpoint may come from the Log and be Trusted On First Use (TOFU).
+Thereafter:
+ 1. The witness acquires a new Checkpoint and a Consistency Proof from its previous state
+ 2. This Consistency Proof is verified, and if the new Checkpoint is confirmed to be Append Only from the previous one, it updates its state
+ 3. The new Checkpoint is countersigned by the Witness, and made available to the public
+
+A feature of [Checkpoints](https://c2sp.org/tlog-checkpoint) is that they allow multiple signatures to be added.
+This means that a common way to see Checkpoints in a witnessed ecosystem is with a log signature first, and then multiple witness signatures.
+An example witnessed checkpoint is:
```

Comment:
This isn't guaranteed by the note format, and I suspect it's only _our_ witness which goes the extra mile to sort signatures in this way.